test/functional/feature_llmq_chainlocks.py

```diff
@@ -66,6 +66,21 @@ def run_test(self):
             block = self.nodes[0].getblock(self.nodes[0].getblockhash(h))
             assert block['chainlock']
 
+        # Update spork to SPORK_19_CHAINLOCKS_ENABLED and test its behaviour
+        self.nodes[0].sporkupdate("SPORK_19_CHAINLOCKS_ENABLED", 1)
+        self.wait_for_sporks_same()
+
+        # Generate new blocks and verify that they are not chainlocked
+        previous_block_hash = self.nodes[0].getbestblockhash()
+        for _ in range(2):
+            self.nodes[0].generate(1)
+            block_hash = self.nodes[0].getbestblockhash()
```

Comment:
could do it in one rpc call
```suggestion
            block_hash = self.nodes[0].generate(1)[0]
```